commands/deploy.go

```diff
@@ -217,14 +223,16 @@ func runDeployCommand(args []string, image string, fprocess string, functionName
 				return envErr
 			}
 
-			// Get FProcess to use from the ./template/template.yml, if a template is being used
-			if languageExistsNotDockerfile(function.Language) {
-				var fprocessErr error
+			if !noReadTemplate {
```

Comment:
I know this is a bit semantic, but the line here is hard to read: _"if not no read template"_

I think it would work better, both for the code, and the user experience, if this variable and flag were named something like "skip-template"

The command would read better: `faas-cli deploy --skip-template`
and this line of code would be easier to understand: _"if not skip template"_